portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructureee/security/SAML.testcase

```diff
@@ -588,59 +588,78 @@ definition {
 		User.viewLoggedOutPG();
 	}
 
-	@description = "This is a use case for LPS-88924"
+	@description = "This is a use case for LPS-88924 using virtual instance"
 	@priority = "4"
 	test AssertSSOWithEncryptionAndDecryption {
 		property test.name.skip.portal.instance = "SAML#AssertSSOWithEncryptionAndDecryption";
 
```

Comment:
Please add property to limit your test case to use only a single tomcat instance via:
property app.server.bundles.size = "0";